src/fe-common/irc/fe-events.c

```diff
@@ -224,7 +224,7 @@ static void event_nick(IRC_SERVER_REC *server, const char *data,
 static void event_mode(IRC_SERVER_REC *server, const char *data,
 		       const char *nick, const char *addr)
 {
-	char *params, *channel, *mode;
+	char *params = NULL, *channel = NULL, *mode = NULL;
```

Comment:
for the moment we try to stay c89 compatible so we cannot initialise them here

Reply:
Ah okay. I've fixed that. Thanks for looking at this!

Reply:
FTR this is actually legal under c89 https://godbolt.org/g/9S3BWR

(but no need to change it back)